docs/layouts/shortcodes/generated/dynamic_section.html

```diff
@@ -86,6 +86,12 @@
             <td>Duration</td>
             <td>Interval at which periodic savepoints will be triggered. The triggering schedule is not guaranteed, savepoints will be triggered as part of the regular reconcile loop.</td>
         </tr>
+        <tr>
+            <td><h5>kubernetes.operator.rescaling.cluster-cooldown</h5></td>
+            <td style="word-wrap: break-word;">1 min</td>
+            <td>Duration</td>
```

Comment:
The default value should be 0 I think to preserve the current behavior as this does not affect all kinds of Kubernetes clusters .

many are actually fixed sized

Reply:
This one is tricky. I would rather like to be conservative here because concurrent scaling operations can create a lot of unexpected churn in the cluster. Users have seen cost increase from resource spikes which can paralyze the entire cluster and lead to adding more k8s nodes (if the cluster autoscaler is active). Users may always change this default. 

Beyond this, we may add something like a concurrency parameter which allows to define how many pipelines are allowed to scale before the cool down. Additionally, checking the actual allocatable resources before scaling is something we need to look into.

Reply:
I personally would not enforce any cross deployment timeout / cooldown by default. I don't think it's intuitive why something on one deployment should have an effect on others. Also this is specific to EKS not a general problem.

While the lack of this config may cause problems on EKS over time, it causes immediate weird behaviour on every operator that is using the autoscaler for multiple jobs. Imagine having 60 jobs, it will take an hour to scale once. Also some jobs may never be scaled because there is no faire queing etc.